truss/remote/baseten/remote.py

```diff
@@ -35,10 +35,17 @@
 
 
 class BasetenRemote(TrussRemote):
-    def __init__(self, remote_url: str, api_key: str, **kwargs):
+    def __init__(
+        self,
+        remote_url: str,
+        api_key: str,
+        remote_base_domain: Optional[str] = None,
```

Comment:
I wonder if we should rename this to `invocation_domain`. `remote_base` seems indistinguishable from the `remote_url` so it's somewhat confusing.

Reply:
agreed that remote and remote_base_domain are too similar, one nit would be to have this be `predict_domain` just to be consistent w/ the name of the endpoint

Reply:
yeah.. I think "base domain" covers the "xyz.com" may be remote_inference_base_domain might be more accurate...